android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Wed Aug 21 14:55:09 IST 2024
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.9-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip
```

Comment:
@KausthubhRzr can you please specify the reason for the downgrade?

Reply:
@vivekshindhe the base branch for my branch is [feat/preftech-customui-ios](https://github.com/razorpay/razorpay-flutter-customui/tree/feat/preftech-customui-ios) in this branch the gradle version was updated, but the master is on [6.7-all.zip](https://github.com/razorpay/razorpay-flutter-customui/blob/turbo-flutter-release/android/gradle/wrapper/gradle-wrapper.properties). Will be picking this up separately.